docs/get-started.md

```diff
@@ -147,6 +147,24 @@ gittuf allows for verifying rules for Git references and files.
 $ gittuf verify-ref --verbose main
 ```
 
+## Verify gittuf itself
+
+You can also verify the state of the gittuf source code repository with gittuf
+itself. At the moment, you'll need to pull down gittuf's metadata manually.
```

Comment:
Could we add a link to the dogfooding doc here, since after this PR https://github.com/gittuf/gittuf/pull/393 is merged in, it will be easier for users to understand how to pull down gittuf specific refs into their repo.